web/src/app/users/UserContactMethodForm.js

```diff
@@ -64,7 +64,7 @@ export default class UserContactMethodForm extends React.PureComponent {
           </Grid>
           <Grid item xs={12}>
             <FormField
-              placeholder='+11235550123'
+              placeholder={`${this.props.value.dialCode}1235550123`}
```

Comment:
it would be great if we could grab the phone number format + default length per the user's locale as well, if possible with an endpoint similar to above

e.g.
United States placeholder -> `+1 555 555 5555`
India placeholder -> `+91 55555 55555`

(not necessarily including the spaces/dashes/parenthesis, that would be out of scope here i believe)

Reply:
Ah I gotcha. Yeah, if we can't find a library that does that for us, I don't think it would be feasible for us to do ourselves